test/test_eval_embedding_retriever.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from haystack.document_stores.elasticsearch import ElasticsearchDocumentStore
+from haystack.nodes.retriever.dense import EmbeddingRetriever
+from haystack.document_stores.memory import InMemoryDocumentStore
+from haystack.nodes.summarizer.transformers import TransformersSummarizer
+from haystack.pipelines import GenerativeQAPipeline, SearchSummarizationPipeline
+from haystack.schema import EvaluationResult
+from test_eval import EVAL_LABELS
```

Comment:
Importing these labels might make it a little bit harder to keep a good overview. For example, somebody might change EVAL_LABELS in test_eval and then wonder why some tests in a different file (test_eval_embedding_retriever.py) fail now. I would have preferred copying the EVAL_LABELS over but I think it's okay. We won't touch the tests too often.

Reply:
should be fixed soon as a second module will not be necessary anymore, see below